charts/seed-controlplane/charts/etcd/values.yaml

```
@@ -62,4 +76,9 @@ limitsRequestsGapScaleParams:
     percentage: 80
   memory:
     value: "5G"
-    percentage: 80
\ No newline at end of file
+    percentage: 80
+prefix: temp 
+storageClass: gardener.cloud-fast
+
+snapshotSchedule: "0 */24 * * *"
+defragmentSchedule: "0 */24 * * *"
```

If this value is for `etcd`, not for `backup`, can you adapt it to be such as
```suggestion
etcd:
  defragmentSchedule: "0 */24 * * *"
```

This makes easier to figure out which value is used for what/where.

---

yes. makes sense.